seed_data/field-guides.json

```diff
@@ -354,7 +354,7 @@
       "  <li>&#8226; Even if you end up using Windows or Mac OSX for development later, your server will almost certainly run Linux, so it's worth getting used to Linux.</li>",
       "  <li>&#8226; Even experienced developers encounter hangups when setting up a development environment. This virtual machine image will remove this tedious process.</li>",
       "</ul>",
-      "<p>Install a bit torrent client, then &thinsp;<a href=\"http://mgnet.me/ZOQk0rd\">download our virtual machine image</a>.</p>",
+      "<p>Install a bit torrent client, for example <a href="http://www.utorrent.com/">utorrent</a> then &thinsp;<a href=\"http://mgnet.me/ZOQk0rd\">download our virtual machine image</a>.</p>",
```

Comment:
Note that there is a syntax error here. Since you're already in double quotes, you must backslash escape these quotes.